source/DSCResources/MSFT_SccmIniFIle/MSFT_SccmIniFIle.psm1

```diff
@@ -0,0 +1,1107 @@
+        Specifies whether to install a CAS or Primary.
+
+    .PARAMETER CDLatest
+        This value informs setup that you're using media from CD.Latest.
+
+    .PARAMETER ProductID
+        Specifies the Configuration Manager installation product key, including the dashes.
+
+    .PARAMETER SiteCode
+        Specifies three alphanumeric characters that uniquely identify the site in your hierarchy.
+
+    .PARAMETER SiteName
+        Specifies the name for this site.
+
+    .PARAMETER SMSInstallDir
+        Specifies the installation folder for the Configuration Manager program files.
+
+    .PARAMETER SDKServer
+        Specifies the FQDN for the server that will host the SMS Provider.
+
+    .PARAMETER PreRequisiteComp
+        Specifies whether setup prerequisite files have already been downloaded.
+
+    .PARAMETER PreRequisitePath
+        Specifies the path to the setup prerequisite files.
+
+    .PARAMETER AdminConsole
+        Specifies whether to install the Configuration Manager console.
+
+    .PARAMETER JoinCeip
+        Specifies whether to join the Customer Experience Improvement Program (CEIP).
+
+    .PARAMETER MobileDeviceLanguage
+        Specifies whether the mobile device client languages are installed.
+
+    .PARAMETER RoleCommunicationProtocol
+        Specifies whether to configure all site systems to accept only HTTPS communication from clients, or to configure the communication method for each site system role.
+
+    .PARAMETER ClientsUsePKICertificate
+        Specifies whether clients will use a client PKI certificate to communicate with site system roles.
+
+    .PARAMETER ManagementPoint
+        Specifies the FQDN of the server that will host the management point site system role.
+
+    .PARAMETER ManagementPointProtocol
+        Specifies the protocol to use for the management point.
+
+    .PARAMETER DistributionPoint
+        Specifies the FQDN of the server that will host the distribution point site system role.
+
+    .PARAMETER DistributionPointProtocol
+        Specifies the protocol to use for the distribution point.
+
+    .PARAMETER AddServerLanguages
+        Specifies the server languages that will be available for the Configuration Manager console, reports, and Configuration Manager objects.
+
+    .PARAMETER AddClientLanguages
+        Specifies the languages that will be available to client computers.
+
+    .PARAMETER DeleteServerLanguages
+        Modifies a site after it's installed. Specifies the languages to remove, and which will no longer be available for the Configuration Manager console, reports, and Configuration Manager objects.
+
+    .PARAMETER DeleteClientLanguages
+        Modifies a site after it's installed. Specifies the languages to remove, and which will no longer be available to client computers.
+
+    .PARAMETER SQLServerName
+        Specifies the name of the server or clustered instance that's running SQL Server to host the site database.
+
+    .PARAMETER DatabaseName
```

Comment:
this could also be instance\<databasename>

Reply:
Added an extra sentence.